### PR TITLE
Removed double hashes from comment's node value

### PR DESCRIPTION
**Pull request by vegansk**
PR for the discussion at http://forum.nim-lang.org/t/1808

### CODE REVIEW

File: compiler/vm.nim [hunk 1]
```diff
@@ -1193,7 +1193,11 @@ proc rawExecute(c: PCtx, start: int, tos: PStackFrame): TFullReg =
       createStr regs[ra]
       let a = regs[rb].node
       if a.kind in {nkStrLit..nkTripleStrLit}: regs[ra].node.strVal = a.strVal
-      elif a.kind == nkCommentStmt: regs[ra].node.strVal = a.comment
+      elif a.kind == nkCommentStmt:
+        var lines = a.comment.split("\n")
+        for idx in 0..high(lines):
+          if lines[idx].startsWith("##"): lines[idx] = lines[idx][2..^0].strip
+        regs[ra].node.strVal = lines.join("\n")
```
Comment (Araq) [inline compiler/vm.nim]: What's wrong with  a.comment.replace("##", "")  ?

Comment (kirbyfan64) [inline compiler/vm.nim]: @Araq Won't that replace double hashes *inside* a comment, too? Like:

```nim
proc x() =
  ## Doc comment: ##
```

Comment (vegansk) [inline compiler/vm.nim]: @Araq, what decision will be made?

**Comment by Araq** [comment]
Devel now strips the ## and the whitespace after the ##. Enjoy!

**Pull request closed without merging**